modules/ztp-install-sno-hardware-reqs.adoc

```diff
@@ -14,6 +14,8 @@ Running vDU application workloads requires a bare-metal host with sufficient res
 |Minimum|4 to 8 vCPU cores|32GB of RAM| 120GB
 |====
 
+Making a change to trigger a Vale check. Don't type OpenShift Container Platform? 
```

Comment:
Use the AsciiDoc attribute '{product-title}' rather than the plain text product term 'OpenShift Container Platform', unless your use case is an exception.